src/environments/gridworlds.jl

```diff
@@ -0,0 +1,26 @@
+export RLBaseGridWorld
+import .GridWorlds
+
+mutable struct RLBaseGridWorld{T<:GridWorlds.AbstractGridWorld} <: RLBase.AbstractEnv
+    env::T
+end
```

Comment:
We can have a field named `state` to cache the agent's view here. So we can call `GridWorld.get_agent_view!(env.state, env.env)` and then return the `env.state` directly in the `RLBase.get_state` function.